### PR TITLE
Test: add test for metric package

### PR DESCRIPTION
**Pull request by SamYuan1990**
It seems github agent removed bcc header files, hence I follow bcc official guide to install bcc headers.
ref to https://github.com/iovisor/gobpf we have to install bcc lib before use it.

Signed-off-by: Sam Yuan <yy19902439@126.com>

### CODE REVIEW

**Comment by SamYuan1990** [comment]
Currently, I am not sure where we added https://github.com/iovisor/gobpf as a build dependency. To make this package been build in success, we have to follow https://github.com/iovisor/gobpf#githubcomiovisorgobpfbcc install https://github.com/iovisor/bcc/blob/master/INSTALL.md#ubuntu---source in github action agent before we run unit test.
That's the reason why our CI fails with check unit test.

File: grafana-dashboards/Kepler-Exporter.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
   "editable": true,
   "fiscalYearStartMonth": 0,
   "graphTooltip": 0,
-  "id": 28,
+  "id": 26,
```
Comment (rootfs) [inline grafana-dashboards/Kepler-Exporter.json:25]: is this change part of this PR?

Comment (SamYuan1990) [reply grafana-dashboards/Kepler-Exporter.json:25]: checking.

Comment (SamYuan1990) [reply grafana-dashboards/Kepler-Exporter.json]: fixed by rebase.

**Comment by SamYuan1990** [comment]
Hence I try to avoid merge myself, but merge it, as approved by @rootfs and it fixed issue in CI.

**Pull request merged**